tutorial_code/example0_1_orbits.py

```diff
@@ -11,26 +11,49 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#
-import dimod
-import networkx as nx
 
+import networkx as nx
 from matplotlib import pyplot as plt
 
+import dimod
+
 from helpers.helper_functions import get_coupler_colors, get_qubit_colors
 from helpers import orbits
 
 
+def plot_orbits(Gnx, pos, bqm):
+    options = {"node_size": 600, "width": 4}
+    nx.draw(
+        Gnx,
+        pos=pos,
+        edge_color=get_coupler_colors(Gnx, bqm=bqm),
+        node_color=get_qubit_colors(Gnx, bqm=bqm),
+        **options,
+    )
+    node_labels = {
+        key: f"{val}" for key, val in nx.get_node_attributes(Gnx, "orbit").items()
+    }
+    nx.draw_networkx_labels(Gnx, pos=pos, labels=node_labels, font_size=14)
+    edge_labels = {
+        key: f"{val}" for key, val in nx.get_edge_attributes(Gnx, "orbit").items()
+    }
+    nx.draw_networkx_edge_labels(Gnx, pos=pos, edge_labels=edge_labels, font_size=14)
+
+
 def make_bqm():
-    """Makes a simple four-spin BQM
+    """Creates a simple model of four connected spins (a Binary Quadratic Model)
+
+    The BQM model represents four "spns" that interact with four spins that interact with their neighbors.
```

Comment:
This sentence sounds wrong. Should it be just 
```suggestion
    The BQM model represents four spins that interact with their neighbors.
```